docs/cloud-soar/cloud-soar-bridge.md

```diff
@@ -133,7 +133,7 @@ The Bridge must be able to resolve DNS hostnames and reach the below destination
    * `SOAR_TOKEN`
 1. Obtain the `SOAR_URL` by clicking **?** at the top of the Cloud SOAR UI and navigating to **API Documentation**. Note the **Servers** value and remove `/v3/` from the end of the URL. The bridge cannot currently be registered to a Cloud SOAR instance with the `/v3/` API. 
 
-And you can set this optional parameter (do not include spaces): `ALIAS`
+And you can set this optional parameter (do not include spaces & less than 20 characters): `ALIAS`
```

Comment:
@mvirga-sumo, I'm not sure what you're saying here. Are you saying, "When you create the parameter, ensure it does not include spaces and is less than 20 characters"?